src/main/java/ui/issuepanel/PanelControl.java

```diff
@@ -267,7 +269,7 @@ private void handleKeys(boolean isForwardKey) {
     }
 
     private void scrollandShowPanel(int selectedPanelIndex, int numOfPanels) {
-        ui.getMenuControl().scrollTo(selectedPanelIndex, numOfPanels);
+        ui.scrollTo(selectedPanelIndex, numOfPanels);
```

Comment:
@dariusf from `ui.scrollTo` to `ui.scrollAndShowPanel`? Also, taking note of below (moving from ui to panelControl) so I will rename and move it to panel control.